cpuminer.go

```diff
@@ -508,3 +509,112 @@ func newCPUMiner(s *server) *CPUMiner {
 		updateHashes:      make(chan uint64),
 	}
 }
+
+// generateNBlocks generates the requested number of blocks. It is self
+// contained in that it creates block templates and attempts to solve them while
+// detecting when it is performing stale work and reacting accordingly by
+// generating a new block template.  When a block is solved, it is submitted.
+// The function returns a list of the hashes of generated blocks.
+func (m *CPUMiner) generateNBlocks(n uint32) ([]*wire.ShaHash, error) {
```

Comment:
ACK, this is done. Also updated rpcserver.go to call the function using the new name.